common/main/af_NA.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
-<!-- Copyright © 1991-2020 Unicode, Inc.
+<!-- Copyright © 1991-2021 Unicode, Inc.
```

Comment:
CLDR-11394 would make this PR clearer: "Modify CLDRModify to only result in changed files". Otherwise we'll see this on the first modify pass of the year.